### ISSUE TITLE
Rubber as paving surface

### ISSUE THREAD

**Issue by RubenKelevra** [westnordost/StreetComplete#1859]
**Use case**
We got pedestrian bridges out of metal with some kind of rubbery surface on them, instead of tarmac.

There seems to be no fitting picture/value for them, currently.

**Proposed Solution**
I guess surface=rubber would fit?

**Picture**

![Screenshot_20200528-191056__01](https://user-images.githubusercontent.com/614929/83172311-b1f27c80-a117-11ea-8978-edbacc001fbe.jpg)

**Comment by westnordost** [comment]
Yes probably, but it is somewhat rare that it is not worth adding a pickable value here, just leave a note

**Comment by RubenKelevra** [comment]
okay

**Comment by PanderMusubi** [comment]
And what about this kind of surface?

https://tntspeeltoestellen.nl/wp-content/uploads/2017/03/Foto-21-06-16-14-43-56-1600x1200.jpg

Is a new type? https://wiki.openstreetmap.org/wiki/Tag:surface%3Dtartan is the closest at the moment.

**Comment by westnordost** [comment]
i think tartan is fine

**Comment by PanderMusubi** [comment]
Perhaps make the example image for StreetComplete show both examples in one image, diagonally split with sports track upper left and playground bottom right, e.g.

**Comment by matkoniecz** [comment]
The example selected was used with intention to fit both.

For reference it is https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/res/drawable-xxhdpi/surface_tartan.jpg

**Comment by RubenKelevra** [comment]
This seems to become more and more common here in my area. I think we need a value for this.

Looking at the established values, "plastic" stands out to me, as well as "[acrylic](https://wiki.openstreetmap.org/wiki/Tag%3Asurface%3Dacrylic)", as [Rubber](https://wiki.openstreetmap.org/wiki/Tag%3Asurface%3Drubber) is already used for playground surfaces from recycled tires.

To be clear: This is not a "bouncy" surface, it's glued directly onto the steel and is as solid as tarmac if you walk on it. It's just to add "texture" to be grippy if wet.

I've seen it also used on concrete indoors, if there's a lot of traffic, as painted concrete can become slippery.

**Comment by matkoniecz** [comment]
it really should be discussed on forum, not here (IMHO neither of this values fits here at all)

**Comment by RubenKelevra** [comment]
surface=plastic works great, what's wrong with that? Looking at the uses, it seems to be used all around the world for exactly for those types of surfaces.

**Comment by mnalis** [comment]
> This seems to become more and more common here in my area. I think we need a value for this.

> Looking at the established values, "plastic" stands out to me, as well as "[acrylic](https://wiki.openstreetmap.org/wiki/Tag%3Asurface%3Dacrylic)", as [Rubber](https://wiki.openstreetmap.org/wiki/Tag%3Asurface%3Drubber) is already used for playground surfaces from recycled tires.

Soooo... perhaps I misunderstand, but is something wrong with existing `surface=acrylic` for marking such "acrylic plastic" surfaces which we already have in StreetComplete? Or is it just a name "[synthetic resin](https://www.ulprospector.com/knowledge/4320/pc-acrylic-resin-fundamentals/)" which is confusing? Or is it the picture? Or something else?

![Image](https://github.com/user-attachments/assets/f06d4ea1-450d-4396-afdc-297a4486211f)

**Comment by matkoniecz** [comment]
> surface=plastic works great, what's wrong with that? 

=plastic seems poor fit for things like

> pedestrian bridges out of metal with some kind of rubbery surface on them

where neither seems to be a plastic

**Comment by RubenKelevra** [comment]
I can't follow you train of thought here. It's made out of plastic obviously and is textured a bit like sandpaper. So surface=plastic works great here. Surface=rubber while it was my first idea, doesn't really fit, as it isn't "rubbery", it's just grippy from the texture.

**Comment by RubenKelevra** [comment]
> > This seems to become more and more common here in my area. I think we need a value for this.
> 
> > Looking at the established values, "plastic" stands out to me, as well as "[acrylic](https://wiki.openstreetmap.org/wiki/Tag%3Asurface%3Dacrylic)", as [Rubber](https://wiki.openstreetmap.org/wiki/Tag%3Asurface%3Drubber) is already used for playground surfaces from recycled tires.
> 
> Soooo... perhaps I misunderstand, but is something wrong with existing `surface=acrylic` for marking such "acrylic plastic" surfaces which we already have in StreetComplete? Or is it just a name "[synthetic resin](https://www.ulprospector.com/knowledge/4320/pc-acrylic-resin-fundamentals/)" which is confusing? Or is it the picture? Or something else?
> 
> ![Image](https://github.com/user-attachments/assets/f06d4ea1-450d-4396-afdc-297a4486211f)

Synthetic resin sounds from the wiki like a surface for sport fields. Can you confirm that it's the same stuff which is being used on pedestrian bridges? 🤔

---

Edit:

I did a bit or research, so DecoTurf etc. which is what "synthetic resin" will tag as surface=acrylic is a completely different product.

The thing used on walkways are sheets of fiberglass reinforced plastic rolled onto the surface and then glued there.

DecoTurf however is a multi layered acrylic product which is mixed on site as a liquid and put on as multiple coats and dried there.

So while DecoTurf and similar might look similar from afar, it's a completely different product, different application and as a sport surface also with a different goal in mind.

So tagging an "acrylic" on ways would be wrong 99% of the time, as its defined as DecoTurf etc. in the wiki.

The closest thing we have in the wiki would be "fibre_reinforced_polymer_grate" which is the same stuff as the sheets just as grates and thicker.

So following this a hypothetical "fibre_reinforced_polymer_plates" would be the most fitting value, if simple "plastic" is not accurate enough for you.

**Comment by mnalis** [comment]
**TL;DR:** probably adding `plastic` is more trouble then benefit, due to its overreaching definition (both generally, but especially in OSM wiki), its overlap with existing values, and more popular OSM values being more precise and providing good compromise.

---

> Synthetic resin sounds from the wiki like a surface for sport fields. Can you confirm that it's the same stuff which is being used on pedestrian bridges? 🤔

What is commonly called ["Plastic"](https://en.wikipedia.org/wiki/Plastic) by common people in non-engineering (e.g. PVC, PET, PP, HDPE, LDPE) is AFAIK likely not very suitable for such surfaces (unless modified with other materials). But the term "plastic" is also over-encompassing, used for huge range of polymers - covering everything from lightweight plastic grocery bags,  sponges, silicon sealants and lubricants, via 3D printing, harder packaging, DVDs and footwear, all the way to Teflon coating on the pans and Kevlar bulletproof body armor. 

So _"plastic"_ is not a particularly good _descriptive_ term IMHO (barely better than calling it _"polymer"_ or _"organic compound"_, which _plastics_ also are).

Now, **_acrylics_ are a _subcategory of plastics_**,  and include things like plexiglas/PMMA and are commonly used for outside, due to its endurance and UV-resistivity. While still a quite wide category, they're significantly narrower than just _"plastic"_.

> I did a bit or research, so DecoTurf etc. which is what "synthetic resin" will tag as surface=acrylic is a completely different product.

It is quite different, yes. Yet _**less different**_ then say grocery bag and Teflon pan, IMHO (which are both covered by the term "plastic"). Also, [DecoTurf](https://en.wikipedia.org/wiki/DecoTurf) is apparently only _partly_ acrylic, having acrylic resin mixed with rubber and silica. (`surface=acrylic` is still a best match for it, lest one invents dozens of new values for various synthetic plastic mixes)

Also, one should not confuse few different (but similar) things:
- `surface=acrylic` (**harder plastic surface**, often used as hardwood replacement for indoors sports like tennis and basketball which need some hardness for efficient ball bouncing, but also for see-though floors on e.g. high tower tourist attractions)  
- `surface=tartan` (somewhat **softer, rubbery-plastic surface**, often used for running tracks, intended to partly absorb foot impacts but still provide some firmness)
- `surface=rubber` (**fully soft natural rubber** or its synthetic counterparts like e.g. silicon rubber, to absorb kinetic energy from kids falling and hurting themselves in playgrounds)
- `surface=artificial_turf` (yet another type of mixed **plastics, but woven into different texture** in order to somewhat simulate natural grass where outdoor sports like e.g. soccer are usually played)
- `surface=carpet` (yet another type of **natural or synthetic plastic fibers, with texture less expressed** than in artificial turf; used for e.g. some corridors)


IMHO, The main problem is that target StreetComplete users are not structural engineers nor material scientists. The another problem is that [ATYL](https://wiki.openstreetmap.org/wiki/ATYL) process (for all its benefits!) unfortunately rarely produces clear and concise definitions, leading to unpreciseness and ambiguity. And third one is that there are tons of variations and mixes, so we should really draw a line somewhere. 

> The closest thing we have in the wiki would be "fibre_reinforced_polymer_grate" which is the same stuff as the sheets just as grates and thicker.

Well, yeah, that would be more specific at least. But it is for grates only (i.e. synthetic `metal_grid` equivalent), and not widely used; and even it were, it would be more  of SCEE then SC material, IMHO.

---

Currently, StreetComplete offers three choices, which is IMHO about ideal for its target audience (_"👨‍💻 Users are no experts:"_):
-  _"Rubber granules"_ for softer elastomers (tags `surface=rubber`, and encompasses both rubber and tartan choices above), and
- _"Synthetic resin"_ for harder synthetic polymers (tags `surface=acrylic`)
- _"Artificial turf"_ (tags `surface=artificial_turf` for grass-like textured synthetic polymers)

Introducing even more generic `surface=plastic` would be counterproductive IMHO. It would both confuse the uses (as more than half of competing choices are also "plastic"), and it would introduce ["Analysis paralysis"](https://en.wikipedia.org/wiki/Analysis_paralysis). Not to mention it would be hard to explain the difference (which [OSM wiki page](https://wiki.openstreetmap.org/wiki/Key:surface) also fails to do).

**Comment by RubenKelevra** [comment]
Well, I think plastic works great, not because it's very detailed, but because it's *not* very detailed. It's the equivalent of "paved".

If the user has no idea what plastic they are looking at, they can just select the group. If the user knows what it is, they can select artificial turf, Synthetic Resin, Rubber granulates, fibre_reinforced_polymer_grate, fibre_reinforced_polymer_plates or whatever the next new surface is. 

> Currently, StreetComplete offers three choices, which is IMHO about ideal for its target audience ("👨‍💻 Users are no experts:")

From the responses here I think some people might actually be misstagging the fibre reinforced polymer plates currently as synthetic resin. Which is worse IMHO than tagging nothing. 

So maybe we should either rename it to acrylic, what the raw value in the wiki is or explain it as "(sport field surface)" to avoid that people gravitate to this selection, regardless if we add plastic as generic term or fibre_reinforced_polymer_plates.

**Comment by mnalis** [comment]
> Well, I think plastic works great, not because it's very detailed, but because it's not very detailed. It's the equivalent of "paved".

That is the most persuasive argument so far. So maybe... But even  _"Plastic (generic)"_ seems too specialized to me. After all, we don't have _"Stones (generic)"_ to cover generic case of stones which are somewhere in between `sett`, `unhewn cobblestone`, `gravel`, `pebbles` & `fine_gravel`; and adding it might cause more confusion and less-specific tagging then it would actually help. 🤷 

> From the responses here I think some people might actually be misstagging the fibre reinforced polymer plates currently as synthetic resin. Which is worse IMHO than tagging nothing.

Why do you think it is _"worse then nothing"_?  [Synthetic resin](https://en.wikipedia.org/wiki/Synthetic_resin) would seem to cover such polymer plates, no? Is it because those polymers are reinforced with extra materials so it is not _"pure"_? (and if so, wouldn't the same argument apply to e.g. tagging reinforced concrete as _"mere"_ `surface=concrete`). Or is it something else why you think it is such a bad answer? Perhaps not clear / understandable enough for layman? Or too generic?

> So maybe we should either rename it to acrylic, what the raw value in the wiki is or explain it as "(sport field surface)" to avoid that people gravitate to this selection

Maybe. I seem to recall there being discussion about texts for those synthetic materials that when they were introduced? Did you find that perhaps in old issues / PRs?

**Comment by RubenKelevra** [comment]
> But even _"Plastic (generic)"_ seems too specialized to me. After all, we don't have _"Stones (generic)"_ to cover generic case of stones which are somewhere in between `sett`, `unhewn cobblestone`, `gravel`, `pebbles` & `fine_gravel`; and adding it might cause more confusion and less-specific tagging then it would actually help. 🤷

I think there's a difference between the stone categories and "plastic".

Plastic is much harder to answer. Anyone who would be asked to specify the surface, can with certainty answer, that "it's some kind of plastic", but they would be hard pressed to answer from the pictures "Rubber granules", ""Synthetic resin", "Artificial turf" or "fibre_reinforced_polymer_plates", as this discussion clearly proofs.

So we inadvertently build a system here which basically requires expert knowledge to answer, because these types of surfaces are hard to seperate from each other.

So my suggestion would be to move the plastic types into a sub category, as they are hard to answer, and have the user just answer "plastic" if they are not certain or if a more precise value is not yet available, as is the case with "fibre_reinforced_polymer_plates".

This makes those surfaces solvable, which avoids that we create a lot of notes for them, but also mark that the surface need more details in the future, like "paved" does.

> > So maybe we should either rename it to acrylic, what the raw value in the wiki is or explain it as "(sport field surface)" to avoid that people gravitate to this selection
> 
> Maybe. I seem to recall there being discussion about texts for those synthetic materials that when they were introduced? Did you find that perhaps in old issues / PRs?

https://github.com/streetcomplete/StreetComplete/pull/5972

> > From the responses here I think some people might actually be misstagging the fibre reinforced polymer plates currently as synthetic resin. Which is worse IMHO than tagging nothing.
> 
> Why do you think it is _"worse then nothing"_? [Synthetic resin](https://en.wikipedia.org/wiki/Synthetic_resin) would seem to cover such polymer plates, no? Is it because those polymers are reinforced with extra materials so it is not _"pure"_? (

Well, the issue is that we show "Synthetic resin" as tag, but tag actually "acrylic" here as raw value. Which is a different type of surface.

It's like tagging asphalt as concrete, because there's not yet an established value for concrete, instead of tagging "paved".

Tagging nothing, or a generic value like "plastic" is better, than tagging the wrong value, because we loose the information, that the value is "not yet precise" and instead say: Yep that's "asphalt" while it is not.

**Comment by westnordost** [comment]
The surface data is used to see which surfaces are acceptable for which mode of transportation and give a rough idea about the quality of the surface (dirt vs asphalt).

I don't have a bad conscience to tag a surface that mimics paving stones but is not made out of stones but e.g. plastic still as paving stones.

Plastic can be a lot of things. If it is an asphalt-like surface, I wonder if it wouldn't even more useful to tag it like that. (If it walks like a duck, and quacks like a duck, ...)

**Comment by matkoniecz** [comment]
> I don't have a bad conscience to tag a surface that mimics paving stones but is not made out of stones but e.g. plastic still as paving stones.

especially as we already have concrete paving stones and stone paving stones

**Comment by mnalis** [comment]
> So my suggestion would be to move the plastic types into a sub category, as they are hard to answer, and have the user just answer "plastic" if they are not certain or if a more precise value is not yet available, as is the case with "fibre_reinforced_polymer_plates".

Did you mean _"sub category"_ here as in:
- building type quest (where e.g. _"Residential"_ category _expands_  to _"detached"_, _"apartments"_, _"house"_ etc. subcategories), or
-  just to somehow describe proposed new `surface=plastic` (in existing **flat** `surface` hierarchy) as _"Some other (generic) type of synthetic plastic polymer, which was unable to be more precisely determined by the mapper if it is soft rubbery granules, synthetic grass-analog, or hard acrylic polymer resin"_ (but obviously, in significantly less words -- like 2-3 max.)
 
> Well, the issue is that we show "Synthetic resin" as tag, but tag actually "acrylic" here as raw value. Which is a different type of surface.

Is it? Perhaps we have different understanding of it. 🤷  IMHO, latter is more precise[^1], but is encompassed by the former, no? But if the naming is the problem, perhaps something like _"Hard plastic-ky material"_ (but more palatable) would be better suited as description for `sufrace=acrylic`? But yes, I can see that name (in particular) could be confusing to some.

> It's like tagging asphalt as concrete

As an aide: I don't think that it was intentional, but what is known by layman as _"asphalt"_ is more properly called ["Asphalt(ic) concrete"](https://en.wikipedia.org/wiki/Asphalt_concrete). So yeah, asphalt is a "kind of" concrete (and more visibly so then, say, reinforced concrete, which might also often get tagged as `surface=concrete`; just as e.g. [bitmac](https://wiki.openstreetmap.org/wiki/Tag:surface%3Dbitmac) might get tagged as less-precise `asphalt`).


[^1]: synthetic resin might also be quite-similarly-looking epoxy-based polymer instead of acrylic-based polymer, but I don't think that is such a big deal for most casual mappers. If it is, they'd probably leave a note with detailed pictures and description of exactly what type it is (or, in SCEE, use raw editor to specify it is `fiberglass_enriched_HDPE` or `fibre_reinforced_polymer_grate` or whatever). But that choice is on the mapper - from SC perspective, they can always answer quite correct (if unspecific) `paved` if they are unsure.

**Comment by matkoniecz** [comment]
Is anyone interested in at least posting on community forum to check what wider community thinks about it? To comfirm that new `surface` value is at least not considered as obviously bad?
